### PR TITLE
Add chain ID to config and export it as a label

### LINKED ISSUE #52: All metrics should include chain_id
Without a chain_id label, it's not easy to silence out blazar alerts when performing maintenance

### PR DESCRIPTION
**Pull request by tschuyebuhl**
### Details
Adds chain ID to config, to help export it as a label. Also adds a sanity check to see if the chain ID we got from Status call matches what's set in the config

### Test plan 
When running with some dummy value for chain ID:
```
7:53PM INF Attempting to get data from /status endpoint with Cosmos RPC client module=blazar package=daemon
Error: failed to initialize daemon: chain ID mismatch, expected: asd, got: deluge-1
Usage:
  blazar run [flags]
```

Otherwise, an example metric:
```
blazar_up{chain_id="deluge-1",compose_file="path-to-docker-compose.yml",hostname="<my-pc-hostname>",version="unknown"} 1
```

### Links
Closes #52

### CODE REVIEW

File: internal/pkg/daemon/daemon_test.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ func TestIntegrationDaemon(t *testing.T) {
 	}()
 
 	// we can't register 2 metrics, but this sharing this should probably cause no problems
-	metrics := metrics.NewMetrics("/path/to/docker-compose.yml", "dummy", "test")
+	metrics := metrics.NewMetrics("/path/to/docker-compose.yml", "dummy", "test", "")
```
Comment (qezz) [inline internal/pkg/daemon/daemon_test.go:78]: i believe we don't check the `metrics` anywhere down the way (i mean in the tests), and providing an empty string to `NewMetrics()` creates a state that should not be possible as per `Config.ValidateAll()`

1/5 i would make it a non-empty string here, just to reflect the more likely state

i would also prefer if we could test the metrics in some way (and make sure it crashes if the `chain_id` is empty),  but i'm not sure if it' possible with the golang prometheus client and `promauto`

**Review by qezz — APPROVED** [review]
:+1:
